amy/fiscal/tests/test_membership.py

```diff
@@ -23,6 +23,8 @@
 )
 from workshops.tests.base import TestBase
 
+CommentModel = django_comments.get_model()
```

Comment:
What is the significance of this change? Do they not return the same model?

Reply:
In our case it's the same model, but there's option to replace the default model with a custom one (much like `User` from `django.contrib.auth`). `.get_model` will return the actually used model.

I decided to switch since I was using mixed approach at some point and wanted to unify.